v8/BUILD.gn

```diff
@@ -1400,6 +1400,10 @@ config("toolchain") {
       # TODO(v8:12245): Fix shadowing instances and remove.
       "-Wno-shadow",
     ]
+    if(is_starboard) {
+      # v8/src/base/debug/stack_trace_posix.cc:68:12: error: unused variable 'kMangledSymbolPrefix' [-Werror,-Wunused-const-variable]
```

Comment:
Move the macro guard in stack_trace_posix.cc instead to only declare those vars when the code is compiled